src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgid "Check your inbox for an email with the confirmation code to enter below:"
+msgstr "Controleer uw inbox voor een e-mail met de bevestigingscode om hieronder in te voeren:"
+
+#: src/screens/StarterPack/Wizard/index.tsx:199
+msgid "Choose Feeds"
+msgstr "Kies Feeds"
+
+#: src/screens/StarterPack/Wizard/index.tsx:195
+msgid "Choose People"
+msgstr "Kies mensen"
+
+#: src/view/com/auth/server-input/index.tsx:76
+msgid "Choose Service"
+msgstr "Kies dienst"
+
+#: src/components/StarterPack/ProfileStarterPacks.tsx:291
+msgid "Choose for me"
+msgstr "Kies voor mij"
+
+#: src/view/com/composer/labels/LabelsBtn.tsx:116
+msgid "Choose self-labels that are applicable for the media you are posting. If none are selected, this post is suitable for all audiences."
+msgstr "Kies labels die van toepassing zijn op de media die u plaatst. Dit bericht is geschikt voor alle doelgroepen als er geen zijn geselecteerd."
+
+#: src/screens/Onboarding/StepFinished.tsx:271
+msgid "Choose the algorithms that power your custom feeds."
+msgstr "Kies de algoritmen die uw aangepaste feeds aansturen."
+
+#: src/screens/Onboarding/StepProfile/AvatarCreatorItems.tsx:107
+msgid "Choose this color as your avatar"
+msgstr "Kies deze kleur als uw avatar"
+
+#: src/screens/Signup/StepInfo/index.tsx:201
+msgid "Choose your password"
+msgstr "Kies uw wachtwoord"
+
+#: src/view/screens/Settings/index.tsx:877
+msgid "Clear all storage data"
+msgstr "Wis alle opslaggegevens"
+
+#: src/view/screens/Settings/index.tsx:880
+msgid "Clear all storage data (restart after this)"
+msgstr "Wis alle opslaggegevens (hierna opnieuw opstarten)"
+
+#: src/components/forms/SearchInput.tsx:70
+msgid "Clear search query"
+msgstr "Wis zoekopdracht"
+
+#: src/view/screens/Settings/index.tsx:878
+msgid "Clears all storage data"
+msgstr "Wist alle opslaggegevens"
+
+#: src/view/com/modals/DeleteAccount.tsx:216
+msgid "Click here for more information."
+msgstr "Klik hier voor meer informatie."
+
+#: src/view/com/modals/DeleteAccount.tsx:208
+msgid "Click here for more information on deactivating your account"
+msgstr "Klik hier voor meer informatie over het deactiveren van uw account"
+
+#: src/components/TagMenu/index.web.tsx:152
+msgid "Click here to open tag menu for {tag}"
+msgstr "Klik hier om het tagmenu voor {tag} te openen"
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:304
+msgid "Click to disable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht uit te schakelen."
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:305
+msgid "Click to enable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht in te schakelen."
+
+#: src/components/dms/MessageItem.tsx:240
+msgid "Click to retry failed message"
+msgstr "Klik om mislukt privébericht opnieuw te proberen"
+
+#: src/screens/Onboarding/index.tsx:32
+msgid "Climate"
+msgstr "Klimaat"
+
+#: src/components/dms/ChatEmptyPill.tsx:39
+msgid "Clip 🐴 clop 🐴"
+msgstr "Clip 🐴 clop 🐴"
```

Comment:
```suggestion
msgstr "Klik 🐴 klak 🐴"
```

I guess???

Reply:
I really don't know. Just copied the suggestion.

Reply:
It was a running joke among the devs and some users to refer to the introduction of DMs using the codename "clip clops" and the 🐴 emoji, so this is just a little easter egg message that is displayed as one of the prompts when starting a new DM conversation with a user :)